code/modules/cargo/packs.dm

```diff
@@ -991,6 +991,14 @@
 	crate_name = "water tank crate"
 	crate_type = /obj/structure/closet/crate/large
 
+/datum/supply_pack/materials/foamtank
+	name = "Foam Tank Crate"
+	desc = "Contains plasmamen's bane."
+	cost = 1500
+	contains = list(/obj/structure/reagent_dispensers/foamtank)
+	crate_name = "foam tank crate"
+	crate_type = /obj/structure/closet/crate/large
+
```

Comment:
I don't mind a couple of these in the CE and Atmos lockers, but I'm not sure this belongs in Cargo. I mean, Chemistry can make this stuff.

Reply:
you cant refill with raw chemicals it needs the tank